src/extension/background-script/NonceService.ts

```diff
@@ -3,29 +3,21 @@ import { web3 } from '../../plugins/Wallet/web3'
 
 OnlyRunInContext(['background', 'debugging'], 'NonceService')
 
-type NonceManagerTask =
-    | { type: 'set'; nonce: number; callback: (e: Error | null) => void }
-    | { type: 'get'; callback: (e: Error | null, nonce: number) => void }
-
 class NonceManager {
     constructor(private address: string) {}
+    #nonce = NonceManager.INITIAL_NONCE
```

Comment:
Is there a good reason to use ECMAScript private field? It's harder to debug in the dev tools. Since we have no other code running or providing this code as a library, I discourage using ES private field.